src/main/java/mServer/crawler/sender/arte/ArteVideoDetailsDeserializer.java

```diff
@@ -0,0 +1,194 @@
+    }
+
+    /**
+     * Liefert den Beginn der Ausstrahlung, 
+     * wenn 
+     *  - heute im Zeitraum von CatchUpRights liegt 
+     *  - oder heute vor dem Zeitraum liegt
+     *  - oder CatchUpRights nicht gesetzt ist und die Ausstrahlung in der Vergangenheit liegt
+     * @param broadcastObject 
+     * @return der Beginn der Ausstrahlung oder ""
+     */
+    private String getBroadcastDate(JsonObject broadcastObject) {
+        String broadcastDate = "";
+        
+        JsonElement elementBegin = broadcastObject.get(JSON_ELEMENT_BROADCAST_CATCHUPRIGHTS_BEGIN);
+        JsonElement elementEnd = broadcastObject.get(JSON_ELEMENT_BROADCAST_CATCHUPRIGHTS_END);
+        
+        if (!elementBegin.isJsonNull() && !elementEnd.isJsonNull()) {
+            String begin = elementBegin.getAsString();
+            String end = elementEnd.getAsString();
+
+            try {
+                Calendar beginDate = Calendar.getInstance();
+                beginDate.setTime(broadcastDateFormat.parse(begin));
+                Calendar endDate = Calendar.getInstance();
+                endDate.setTime(broadcastDateFormat.parse(end));
+
+                if(DateWithoutTimeComparer.compare(today, beginDate) >= 0 && DateWithoutTimeComparer.compare(today, endDate) <= 0) {
+                    // wenn das heutige Datum zwischen begin und end liegt,
+                    // dann ist es die aktuelle Ausstrahlung
+                    broadcastDate = broadcastObject.get(JSON_ELEMENT_BROADCAST).getAsString();
+                } else if(DateWithoutTimeComparer.compare(today, beginDate) < 0) {
+                    // ansonsten die zukünftige verwenden
+                    broadcastDate = broadcastObject.get(JSON_ELEMENT_BROADCAST).getAsString();
+                }
+                
+            } catch (ParseException ex) {
+                LOG.debug(ex);
+            }           
+        } else {
+            String broadcast = broadcastObject.get(JSON_ELEMENT_BROADCAST).getAsString();
+            
+            try {
+                Calendar broadcastCal = Calendar.getInstance();
+                broadcastCal.setTime(broadcastDateFormat.parse(broadcast));
+                broadcastDate = broadcast;
+                
+            } catch (ParseException ex) {
+                LOG.debug(ex);
+            }            
+        }
+        return broadcastDate;
+    }    
+    
+    private static String getFirstBroadcastDateIgnoringCatchupRights(JsonArray broadcastArray) {
+        String broadcastDate = "";
+        
+        for(int i = 0; i < broadcastArray.size(); i++) {
+            JsonObject broadcastObject = broadcastArray.get(i).getAsJsonObject();
+            
+            if(broadcastObject.has(JSON_ELEMENT_BROADCASTTYPE) && 
+                broadcastObject.has(JSON_ELEMENT_BROADCAST)) {
+                String type = broadcastObject.get(JSON_ELEMENT_BROADCASTTYPE).getAsString();
+                
+                switch(type) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)